client_sdk/avalon_client_sdk/interfaces/work_invocation_interface.py

```diff
@@ -0,0 +1,55 @@
+from abc import ABC,abstractmethod
```

Comment:
Add Apache 2.0 license.

Reply:
Updated in the latest commit.Please review